### DISCUSSION TITLE
Frontend

### DISCUSSION THREAD

**Discussion by ZompaSenior** [ConoscereLinux/asso-django#3] [General]
It is probably a bit premature, but in the meantime I put the question on the table: do we start with a pure Bootstrap or with some template or extension?
I am for a pure Bootstrap: we have no particular graphic needs and so we do something simple, and when necessary we always have our hands free.

**Comment by giobber** [comment]
It is a bit premature, but is an important topic for the future
I'm not really sure of what you intend for "template or extension"

In the last 2-3 years I've used mainly [tachyons](https://tachyons.io/) as my CSS framework of reference, very lightweight and intuitive, optimal to build up a personal site with a modern and customized feel. If I want to be fast on developing the site's style A'll pick this one.

On the other side, **Bootstrap** has a lot of useful features, in particular **SCSS theming** (which is a feature that I absolutely want in this project). Other advantages of using bootstrap in a public project is his capillary diffusion in web projects

A 'must-of' in any case will be a SCSS compiler, I think there are various library to build up SCSS automatically on django server restart, at the moment `django_compressor` and `django-libsass` seem to be promising

**Reply by ZompaSenior** [reply]
I think we are aligned.
Regarding the SCSS, I'm a little immature about it, but I intend to update myself on the matter.

**Reply by giobber** [reply]
For simply using bootstrap without changing its defaults SCSS is not necessary

But if you want ti change his defaults without using bad and ugly CSS rules with `!important` everywhere...that Is the way
